Tasks/ServiceFabricDeploy/make.json

```diff
@@ -1,4 +1,18 @@
 {
+    "externals": {
+        "nugetv2": [
+            {
+                "name": "Microsoft.IdentityModel.Clients.ActiveDirectory",
+                "version": "2.23.302261847",
```

Comment:
this is the version that the agent previously carried. the agent moved to 3.x and busted the fragile assumption of the task. with this change, the required version of the dll will be bundled with the task.